cognite/powerops/custom_modules/power_model_v1/data_models/views/shop_result/SHOPResultPriceProd.view.yaml

```diff
@@ -0,0 +1,45 @@
+space: '{{powerops_models}}'
+externalId: SHOPResultPriceProd
+name: SHOPResultPriceProd
+description: A shop result type that extends the generic shop result with a direct link to the price and production time series
+filter:
+  and:
+  - hasData:
+    - type: container
+      space: '{{powerops_models}}'
+      externalId: SHOPResult
+  - equals:
+      property:
+      - node
+      - space
+      value: '{{powerops_instance_space}}'
+  - equals:
+      property:
+      - node
+      - type
+      value:
+        externalId: SHOPResultPriceProd
+        space: '{{powerops_type_space}}'
+implements:
+- space: '{{powerops_models}}'
+  externalId: SHOPResult
+  version: '{{version}}'
+  type: view
+version: '{{version}}'
+properties:
+  priceTimeseries:
+    container:
+      space: '{{powerops_models}}'
+      externalId: SHOPResult
+      type: container
+    containerPropertyIdentifier: tsDirect1
+    name: tsDirect1
+    description: The market price timeseries from the Shop run
+  productionTimeseries:
+    container:
+      space: '{{powerops_models}}'
+      externalId: SHOPResult
+      type: container
+    containerPropertyIdentifier: tsDirectList1
+    name: tsDirectList1
+    description: The production timeseries for each plant included in the Scenario from the Shop run
```

Comment:
Nice one. Are the time series somehow linked to which plant they belong to?

Reply:
I can add that. That would require an intermediate step between a general ShopResult and the type from this view SHOPResultPriceProd